Gemfile

```diff
@@ -41,6 +41,7 @@ group :development do
 end
 
 group :test do
+  gem 'debug'
```

Comment:
Can you say more about how `debug` relates to e.g. `byebug`? I wonder if `pry-byebug` can be removed now?